src/components/Meetups/2024/Agenda/index.tsx

```diff
@@ -57,7 +59,7 @@ export default function Agenda({ title, subtitle, lastUpdate, agenda }: AgendaPr
                     <div className="flex w-full flex-row items-center justify-between gap-3 text-sm md:text-lg">
                       <div className="flex flex-row items-center justify-between gap-3">
                         <span className="text-yellow-400 md:mr-3">
-                          {format(startTime, "HH:mm")} - {format(endTime, "HH:mm")}
+                          {format(new UTCDate(startTime), "HH:mm")} - {format(new UTCDate(endTime), "HH:mm")}
```

Comment:
Is the `UTCDate` really required?

Reply:
The server was originally returning UTC dates, but I updated the configuration, so it's no longer necessary!